### ISSUE TITLE
BasicSocket#recvmsg and Socket::MSG_ERRQUEUE

### ISSUE THREAD

**Issue by ippa** [oneclick/rubyinstaller#48]
TL;DR; Why isn't @socket.recvmsg and Socket::MSG_ERRQUEUE to get info out of a ICMP unreachable when sending UDP to nonlistening host available in latest windows ruby?
Posting here I think it might be a buildissue, but I honestly have no idea :).

A bit longer description of the question is available here: 
http://stackoverflow.com/questions/4567667/how-do-i-get-the-ip-of-incoming-icmp-due-to-udp-send-to-dead-client-in-ruby

As the answer points out, I want to use this: http://homepages.cwi.nl/~aeb/linux/man2html/man2/recvmsg.2.html .. but that's not possible in windows ruby.

The relevant recvmsg()-stuff seems to be located in https://github.com/ruby/ruby/blob/trunk/ext/socket/ancdata.c

**Comment by marek22k** [comment]
@ippa Have you found a solution to the problem? I have the same problem on Linux.
